kin-sdk/kin-sdk-lib/src/androidTest/kotlin/kin/sdk/internal/queue/PaymentQueueManagerIntegrationTest.kt

```diff
@@ -0,0 +1,165 @@
+package kin.sdk.internal.queue
```

Comment:
Should be removed and integration  be added after finishing both queues (payment and tasks) right?

Reply:
Correct